tails_files/install.sh

```diff
@@ -74,7 +74,7 @@ Host mon
 EOL
     chown amnesia:amnesia $INSTALL_DIR/ssh_config
     chmod 600 $INSTALL_DIR/ssh_config
-    cp -p $INSTALL_DIR/ssh_config $HOMEDIR/.ssh/config
+    cp -pf $INSTALL_DIR/ssh_config $HOMEDIR/.ssh/config
```

Comment:
I figure since it is also prepared and modified in place (adding the hostname and username) then it makes sense to have it in the install dir. If you don't feel that's apt then we could change it to edit the .ssh/config directly.